### ISSUE TITLE
Can't load node modules when app is in non-ascii path

### ISSUE THREAD

**Issue by zh99998** [systemjs/systemjs#1512]
I'm building a electron app and find that if app is in a non-ascii path (for example `D:\Github\mycard - 副本`), it will fail to load node modules.

my system.config.js:

```js
...
"aria2": "@node/aria2",
...
```

my code:
```js
const aria2 = require('aria2')
```

and got this error:
https://lh3.googleusercontent.com/-L9kgoPvIenk/WD5WR6-2cKI/AAAAAAAAFHc/ixK4vRmfCgMYR00imf9y-O9Y8HQwr06ggCL0B/h1200/2016-11-29.png
![](https://lh3.googleusercontent.com/-L9kgoPvIenk/WD5WR6-2cKI/AAAAAAAAFHc/ixK4vRmfCgMYR00imf9y-O9Y8HQwr06ggCL0B/h1200/2016-11-29.png)

it tried to load module from `file:///D:/Github/mycard%20-%20%E5%89%AF%E6%9C%AC/`, the encoded url. until here it's correct.
but later module.js try to load `D:/Github/mycard%20-%20%E5%89%AF%E6%9C%AC/` from `fs` without url decode, and fail.

https://lh3.googleusercontent.com/-F16l0ZoycW0/WD5WS-6RG3I/AAAAAAAAFHg/nRhF0anPoD09Y42uZYuRclJI3rDxTuAzgCL0B/h1200/2016-11-29.png
![](https://lh3.googleusercontent.com/-F16l0ZoycW0/WD5WS-6RG3I/AAAAAAAAFHg/nRhF0anPoD09Y42uZYuRclJI3rDxTuAzgCL0B/h1200/2016-11-29.png)

**Comment by guybedford** [comment]
Ah right, I think this will be fixed in the next 0.20 alpha version of SystemJS. Perhaps try that release in the mean time?

**Comment by guybedford** [comment]
I added a test for this for the SystemJS 0.20 release candidate in https://github.com/systemjs/systemjs/commit/98e05c0bc89d14790e2b5f463e87b332911219cf, and it is passing correctly.
